reviews three/swagger.yaml

```diff
@@ -0,0 +1,954 @@
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: Draft Orders API
+  x-logo:
+    url: "https://andrewjtech123.github.io/imagehost/shopify_logo_white.png"
+    backgroundColor: "#7ab55c"
+  description: >
+    # Draft Orders API 
+      
+      A [Draft Order](https://help.shopify.com/manual/orders/create-orders) represents the status of an order before the customer has completed the checkout process. Using the Draft Orders API you can create a draft order and transition it to a completed order with great flexibility.  The following flows are supported:
```

Comment:
A Draft Order represents the status of an order before the order is set to completed status.